angular/src/directives/input-strip-spaces.directive.ts

```diff
@@ -0,0 +1,12 @@
+import { Directive, ElementRef, HostListener } from "@angular/core";
+
+@Directive({
+  selector: "input[appChangeStripSpaces]",
```

Comment:
The name seems a bit off, why is Change included in the directive? `appInputStripSpaces` seems cleaner, or simply `appStripSpaces`.

Reply:
Changed to `appInputStripSpaces`, that follows the convention of the other directives.